README.md

```diff
@@ -19,10 +19,22 @@ ESLint configuration file, e.g. 📄 .eslintrc.js:
 {
   ...,
   "plugins": [ ... ],
-  "extends": "@digital-h/eslint-config" <---
+  "extends": "@digital-h/eslint-config", <---
   "rules": [...]
 }
 ```
+For TypeScript projects you might need to add tsconfig to parserOptions:
+```
+{
+  ...,
+  "plugins": [ ... ],
+  "extends": "@digital-h/eslint-config",
+  "parserOptions: {
```

Comment:
Ja macht Sinn, in den samples ist das ähnlich mit drin, ich packe noch einen Satz dazu, dass man sich die auch noch anschauen kann